xgcm/grid_ufunc.py

```diff
@@ -555,7 +586,10 @@ def _rechunk_to_merge_in_boundary_chunks(
 
 
 def _get_chunk_pattern_for_merging_boundary(
-    grid, da, original_chunks, boundary_width_real_axes
+    grid: "Grid",
```

Comment:
```suggestion
    grid: Grid,
```

I've not seen these quoted class names before. Is that related to the circular import?

Reply:
Yes it's allows for a form of delayed evaluation of the type IIUC, but I was able to remove this - see [comment above](https://github.com/xgcm/xgcm/pull/380#discussion_r762286826).